plugins/modules/identity/ipa/ipa_otptoken.py

```diff
@@ -0,0 +1,530 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2017, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: ipa_otptoken
+author: justchris1 (@justchris1)
+short_description: Manage FreeIPA OTPs
+version_added: 2.5.0
+description:
+- Add, modify, and delete One Time Passwords in IPA.
+options:
+  uniqueid:
+    description: Unique ID of the token in IPA.
+    required: True
```

Comment:
```suggestion
    required: true
```
All YAML boolean should be lower-case `true`/`false`.